### PR TITLE
added abnumber lib requirement, ablang api, modified clustered.tsv format

### PR DESCRIPTION
**Pull request by YoelShoshan**

### CODE REVIEW

**Comment by michalozeryflato** [comment]
@YoelShoshan I see that the tests failed - please check.
Also, can you please add to this PR the following changes in requirements.txt:
- **openfold** package => point to internal fork for fixing numpy issue https://github.ibm.com/BiomedSciAI-Innersource/openfold
- add **biopython** package
Thanks!

**Comment by mosheraboh** [comment]
@YoelShoshan what is the status of this PR?

**Comment by YoelShoshan** [comment]
> @YoelShoshan I see that the tests failed - please check. Also, can you please add to this PR the following changes in requirements.txt:
> 
>     * **openfold** package => point to internal fork for fixing numpy issue https://github.ibm.com/BiomedSciAI-Innersource/openfold
> 
>     * add **biopython** package
>       Thanks!

Sorry, I only noticed your comment just now.

for openfold I think you already added the one from my public github, I'll check
same for biopython

File: fusedrug/data/protein/structure/structure_io.py [hunk 1]
```diff
@@ -220,10 +236,15 @@ def get_chain_native_features(
...
         # move to device
-        gt_mmcif_feats = {k: gt_all_mmcif_feats[k] for k in ["aatype", "all_atom_positions", "all_atom_mask"]}
+        gt_mmcif_feats = {
+            k: gt_all_mmcif_feats[k] for k in ["aatype", "all_atom_positions", "all_atom_mask", "resolution"]
+        }
 
         to_tensor = lambda t: torch.tensor(np.array(t)).to(device)
         gt_mmcif_feats = tree_map(to_tensor, gt_mmcif_feats, np.ndarray)
+
+        if gt_mmcif_feats["resolution"].shape[0] == 0:
```
Comment (mosheraboh) [inline fusedrug/data/protein/structure/structure_io.py:246]: ?
banana filter 😆

File: fusedrug/utils/cluster/cluster_using_mmseqs.py [hunk 2]
```diff
@@ -180,6 +179,13 @@ def cluster(
     cmd = f"mmseqs createtsv {step_B_initial_db} {step_B_initial_db} {clustered_db} {clustered_tsv}"
     _run_system_cmd(cmd)
 
+    # sort it to avoid issues with people assuming it's sorted
```
Comment (mosheraboh) [inline fusedrug/utils/cluster/cluster_using_mmseqs.py:182]: Maybe also check that it has the expected size?

**Review by mosheraboh — APPROVED** [review]
LGTM
See inline

**Review by mosheraboh — APPROVED** [review]
LGTM
See inline

File: fusedrug/data/protein/structure/structure_io.py (hunk 1 above)
Comment (YoelShoshan) [reply fusedrug/data/protein/structure/structure_io.py:246]: woops - deleting :)

**Pull request merged**